loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -200,38 +205,38 @@ static int binary_update_download_new_binary(void)
 	/* Get 'micom' binary file size and check it with available size */
 	filesize = lseek(read_fd, 0, SEEK_END);
 	if (filesize <= 0 || bin_info.available_size <= 0 || filesize >= bin_info.available_size) {
-		fail_cnt++;
 		printf("Can't copy file, size %d, available size %d in fs\n", filesize, bin_info.available_size);
-		return ERROR;
+		ret = ERROR;
```

Comment:
I think we'd better initialize the `ret` as `ERROR` and assign `OK` when success case. Then we can reduce much code.

Reply:
A 'ret' variable is used for return value of functions many times so I set ERROR when each failures.